three-nodes/docker-compose.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
How about delete this file? It's out of date, confusing to mix ansible and docker in the same directory, and there's a pending standalone repo: https://github.com/manics/docker-example-omero/tree/master